core/modules/moduleconf.py

```diff
@@ -30,19 +46,26 @@ class ModuleConf(List):
     kindName = "viur-module-conf"
     accessRights = ["edit"]
 
+    def adminInfo(self):
+        admin_info = super().adminInfo()
+        admin_info["icon"] = "icon-settings"
+        admin_info["display"] = "hidden"
+        return admin_info
+
     def canAdd(self):
         return False
 
     def canDelete(self, skel: SkeletonInstance) -> bool:
         return False
 
     @classmethod
-    def get_by_module_name(cls, module_name: str):
-        db_key = db.Key("viur-module-conf", f"{module_name}")
-        skel = getattr(conf["viur.mainApp"], "_moduleconf").viewSkel()
+    def get_by_module_name(cls, module_name: str) -> Union[None, SkeletonInstance]:
```

Comment:
Please add the `Union` import as [recommended](https://github.com/viur-framework/viur-core/pull/577#discussion_r1041903602) by @ArneGudermann 

If you don't like imports😅 you could write:
```suggestion
    def get_by_module_name(cls, module_name: str) -> None | SkeletonInstance:
```

Reply:
> If you don't like imports

I hate imports...